.dockerignore

```diff
@@ -1,2 +1,2 @@
 *
-!ci/
+!docker/
```

Comment:
Surely this commit can just be removed, along with the same commit that renamed this stuff in the first place? :-)